src/cc/tests/environments/ChunkserverEnvironment.cc

```diff
@@ -26,18 +28,21 @@ ChunkserverEnvironment::GenerateConfig()
     mClientPort = QFSTestUtils::GetRandomPort(SOCK_STREAM, "127.0.0.1");
     mChunkDir = QFSTestUtils::CreateTempDirectory();
 
+    map<string, string> c;
+    c["chunkServer.metaServer.hostname"] = "localhost";
+    c["chunkServer.metaServer.port"] = lexical_cast<string>(mMetaserverPort);
+    c["chunkServer.clientPort"] = lexical_cast<string>(mClientPort);
+    c["chunkServer.chunkDir"] = mChunkDir;
+    c["chunkServer.clusterKey"] = "some-random-unique-identifier";
+    c["chunkServer.stdout"] = "/dev/null";
+    c["chunkServer.stderr"] = "/dev/null";
+    c["chunkServer.msgLogWriter.logLevel"] = "INFO";
+
     string config;
-    config += "chunkServer.metaServer.hostname = localhost\n";
-    config += StrUtils::str_printf("chunkServer.metaServer.port = %d\n",
-            mMetaserverPort);
-    config += StrUtils::str_printf("chunkServer.clientPort = %d\n",
-            mClientPort);
-    config += StrUtils::str_printf("chunkServer.chunkDir = %s\n",
-            mChunkDir.c_str());
-    config += "chunkServer.clusterKey = some-random-unique-identifier\n";
-    config += "chunkServer.stdout = /dev/null\n";
-    config += "chunkServer.stderr = /dev/null\n";
-    config += "chunkServer.msgLogWriter.logLevel = INFO\n";
+    for (map<string, string>::const_iterator itr = c.begin(); itr != c.end();
+            ++itr) {
+        config += itr->first + " = " + itr->second + "\n";
+    }
```

Comment:
I am probably missing something, but why don't we do the following directly?
config += "chunkServer.metaServer.hostname = " + lexical_cast<string>(mMetaserverPort) + "\n";
...